### PR TITLE
add note about wallet addresses to layer 2 page

### PR DESCRIPTION
**Pull request by corwintines**
## Description
- Add a note about EOA and smart contract wallets on layer 2

### CODE REVIEW

File: src/pages/layer-2.js [hunk 1]
```diff
@@ -679,6 +679,15 @@ const Layer2Page = ({ data }) => {
         <p>
           <Translation id="layer-2-use-layer-2-1" />
         </p>
+        <p>
+          NOTE: When bridging over and using layer 2, it is important to note
+          that you will control the address for you EOA account just like on
+          Ethereum Mainnet. However, if you are using a smart contract wallet,
```
Comment (timbeiko) [inline src/pages/layer-2.js:685]: I would link a couple examples here, such as Gnosis Safe, Argent, etc.

File: src/pages/layer-2.js [hunk 2]
```diff
@@ -679,6 +679,15 @@ const Layer2Page = ({ data }) => {
         <p>
           <Translation id="layer-2-use-layer-2-1" />
         </p>
+        <p>
+          NOTE: When bridging over and using layer 2, it is important to note
+          that you will control the address for you EOA account just like on
```
Comment (timbeiko) [inline src/pages/layer-2.js:684]: I would expand on "EOA" account, e.g. an account where only a single private key controls the funds / users a recovery phrase / etc. given that this warning is probably more for novice than advanced users.

File: src/pages/layer-2.js [hunk 3]
```diff
@@ -679,6 +679,15 @@ const Layer2Page = ({ data }) => {
...
+          Ethereum Mainnet. However, if you are using a smart contract wallet,
+          you will not have control over this address on a layer 2 by default
+          unless deployed using `CREATE2`. If you are bridging or sending funds
```
Comment (timbeiko) [inline src/pages/layer-2.js:687]: > unless deployed using `CREATE2`

wouldn't you need to also redeploy on the L2 before? If there is an extra step to take we should maybe list it or simplify the sentence.

**Review by timbeiko — COMMENTED** [review]
Thanks for getting this up so quickly! Added a couple minor comments 😄

**Review by minimalsm — APPROVED** [review]

**Pull request merged**